tests/unit/test_database.py

```diff
@@ -2317,6 +2341,48 @@ def test_generate_read_batches_w_data_boost_enabled(self):
             "keyset": {"all": True},
             "index": self.INDEX,
             "data_boost_enabled": True,
+            "directed_read_options": None,
```

Comment:
We are adding directed read options in databoost test. If its intentional, rename the test .

Reply:
As discussed offline, the request assertion fails so we need to have this with its default value.